src/create-store.js

```diff
@@ -21,7 +21,7 @@ export function createStore(model, options = {}) {
   const modelClone = clone(model);
   const {
     compose,
-    devTools = process.env.NODE_ENV !== 'production',
```

Comment:
Any type updates we need to consider here?
Should we support both bool and object input?

Reply:
Hmmm. Good point. Will have to review.

Reply:
I think I've fixed it now - but I was unable to verify it 😕 

Tried to install `easy-peasy` by using `file:../../` as the version for the example - but I just got the `5.1.0` source. And yes, I did run `yarn build` before doing this.